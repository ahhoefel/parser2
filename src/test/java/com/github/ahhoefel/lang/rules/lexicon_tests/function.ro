func main() int {
    return 3
}