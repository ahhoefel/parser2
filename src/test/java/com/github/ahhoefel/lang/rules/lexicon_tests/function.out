func(func) @ :1:1:4
identifierGrammar(main) @ :1:6:4
lparen(() @ :1:10:1
rparen()) @ :1:11:1
int(int) @ :1:13:3
lbrace({) @ :1:17:1
return(return) @ :2:5:6
numberGrammar(3) @ :2:12:1
rbrace(}) @ :3:1:1
